bitwarden_license/src/Scim/Users/GetUserQuery.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using Bit.Core.Exceptions;
```

Comment:
Let's just pull this `GetUserQuery` back out into the controller (i.e. leave it how it was). It was an early one that I approved before we had that discussion with Oscar about when it's worth using a query. In this case, the repository call is basically the query, there's so little code here that we can just have it in the controller. (sorry for the busywork)

Reply:
You can do this in a separate PR though, if you want.

Reply:
I'll do that on the next PR when merging to master. Thanks!